src/ws-date-picker/ws-date-picker.scss

```diff
@@ -263,7 +264,7 @@
     font-size: rem(14px);
     width: rem(32px);
     height: rem(32px);
-    line-height: rem(32px);
+    line-height: rem(28px); // Numbers on acumin-pro font are lower
```

Comment:
If you tell me how to calculate the line height for this specific font I gonna use it ;) :D